internal/sirius/add_payment.go

```diff
@@ -7,16 +7,10 @@ import (
 	"net/http"
 )
 
-func (c *Client) AddPayment(ctx Context, caseID int, amount int, source string, paymentDate DateString) error {
-	postData, err := json.Marshal(struct {
-		Amount      int        `json:"amount"`
-		Source      string     `json:"source"`
-		PaymentDate DateString `json:"paymentDate"`
-	}{
-		Amount:      amount,
-		Source:      source,
-		PaymentDate: paymentDate,
-	})
+const FeeReductionSource = "FEE_REDUCTION"
+
+func (c *Client) AddPayment(ctx Context, caseID int, amount int, source string, paymentDate DateString, feeReductionType string, paymentEvidence string, appliedDate DateString) error {
```

Comment:
Again I think we should separate these out and have a `add_fee_reduction.go` in both `internal/sirius` and `internal/server`, as the only common field used in adding payments and fee reductions is `source` theres not much point in combining them. Therefore I think we should also create a new endpoint in sirius to add fee reductions

Reply:
Happy to separate the handlers as I agree it could get messier with future work however, with the endpoints the plan is to reuse the payments endpoints with conditional logic for the different types of payments so we can easily perform calculations in the future